ev2/event-driven/algs/surface.h

```diff
@@ -270,6 +272,24 @@ class SCARF
         if(rfs.size()) return rfs[0].N;
         else return -1;
     }
+
+    /**
+     * @brief Get the Scarf Params object
+     * 
+     * @return std::tuple<cv::Size count, cv::Size dims, double border_size>
+     *         count        : Resolution of SCARF blocks
+     *         dims         : Size of receptive field
+     *         border_shift : (HARD CORDED) pixel values for shift size to match origin of image and origin of scarf
+     */
+    std::tuple<cv::Size, cv::Size, cv::Size> getScarfParams(void)
```

Comment:
what about returning N also with getScarfParams() so we only have 1 function to get all parameters?